### ISSUE TITLE
0.8.1 Snap package fails to launch on Ubuntu 20.04

### ISSUE THREAD

**Issue by LordObsidian01** [flameshot-org/flameshot#980]
**Flameshot version**
V0.8.1 Snap package

**Describe the bug**

**To Reproduce**
```
wget https://github.com/flameshot-org/flameshot/releases/download/v0.8.1/flameshot_v0.8.1_amd64.snap
sudo snap install --dangerous flameshot_v0.8.1_amd64.snap
flameshot gui 
# nothing happens
```

**Expected behavior**
I expect the gui to launch to take a screen clip. I can run just `flameshot` and it launches the system tray icon but cannot interact with it. `flameshot full -c` does not store anything to the clipboard as well. 

**System Information**
Ubuntu 20.04 X11

**Comment by borgmanJeremy** [comment]
Thanks, I confirmed this in a VM. I think our snap is missing some permissions.

**Comment by LordObsidian01** [comment]
Ok, I can confirm this behavior as well. I was not able to see this before when using i3 as I do not have the tray icon working there. The notification icon does not have a drop down when I perform a single click, however. A double click will launch the capture function. I have a video attached, you cannot see my single clicking but I promise I am!

The command `flameshot gui` however, does nothing (second video)
[screen.zip](https://github.com/flameshot-org/flameshot/files/5286895/screen.zip)

**Comment by borgmanJeremy** [comment]
## Summary of Issues

* ~~Snap does not launch~~
This is just a long delay when first launching the snap

* ~~Flameshot gui does not work~~
This is unrelated to being a snap, I believe it is a duplicate of #790. So a real bug, but not unique to snap

* Menu does not work
This is a confirmed issue with the snap. When launching with --devmode the issue goes a way so we are missing a permission. After fixing this I will close this issue.

**Comment by borgmanJeremy** [comment]
This should be resolved by #992 . Can you test the snap from the github action: https://github.com/flameshot-org/flameshot/pull/992/checks?check_run_id=1171314184

direct link: https://we.tl/t-uwQlHIWOHU

**Comment by borgmanJeremy** [comment]
Merged in fix and published fix to snapstore.
